software/cooler/zoomify/functions.nf

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
This is the old `functions.nf` file. Be great if you can update this for all the current PRs.